packages/chrono-mongo-datastore/src/chrono-mongo-datastore.ts

```diff
@@ -253,6 +270,62 @@ export class ChronoMongoDatastore<TaskMapping extends TaskMappingBase>
     return this.toObject(task);
   }
 
+  /**
+   * Add a task to the Dead Letter Queue
+   *
+   * @param task The task to move to DLQ
+   * @param error Optional error that caused the task to fail
+   */
+  async addToDlq<TaskKind extends keyof TaskMapping>(
+    task: Task<TaskKind, TaskMapping[TaskKind]>,
+    error?: Error,
+  ): Promise<void> {
+    const database = await this.getDatabase();
+    const dlqName = this.config.dlqCollectionName ?? 'chrono-tasks-dlq';
+    const dlqCollection = database.collection(dlqName);
+    const mainCollection = database.collection(this.config.collectionName);
+
+    // Convert task.id (string) back to ObjectId
+    const objectId = new ObjectId(task.id);
+
+    // Insert into DLQ using _id
+    await dlqCollection.insertOne({
+      ...task,
+      _id: objectId,
+      error: error?.message,
+      failedAt: new Date(),
+    });
+
+    // Remove from main collection by _id
+    await mainCollection.deleteOne({ _id: objectId });
+  }
+
+  /**
+   * Redrive messages from the Dead Letter Queue back into main store
+   */
+  async redriveFromDlq<TaskKind extends keyof TaskMapping>(): Promise<void> {
+    const database = await this.getDatabase();
+    const dlqName = this.config.dlqCollectionName ?? 'chrono-tasks-dlq';
```

Comment:
defaults should be decide in the constructor. aka we can make dlqCollectionName required